linkcheck/bookmarks/safari.py

```diff
@@ -23,6 +25,9 @@
 except ImportError:
     has_biplist = False
 
+if sys.version_info >= (3, 9):
+    has_biplist = False
```

Comment:
This just cries for a comment explaining why we're setting `has_biplist = False` even if `import biplist` succeeded on Python 3.9+.

Reply:
I mean one can dig out that information from git commit messages, but.

Also, that comment should include a link to a biplist bug report about the Python 3.9 compatibility problem, so we can click it and go see when it gets fixed and remove the override in the future.

Reply:
in general, what i sometimes do here is:

```suggestion
try:
    import biplist
except ImportError:
    biplist = None
```

that avoids the extra variable. but that's somewhat unrelated. :)

Reply:
Filed a bug and added a link in our code.